jerry-core/api/jerry-debugger.c

```diff
@@ -116,3 +116,63 @@ jerry_debugger_cleanup (void)
   }
 #endif /* JERRY_DEBUGGER */
 } /* jerry_debugger_cleanup */
+
+/**
+ * Sets whether the engine should wait for a source.
+ */
+void
+jerry_debugger_wait_source (void)
```

Comment:
I feel the name is not the best (it actually runs the code, not just wait for it), but unsure about a better name. Some ideas:

`jerry_debugger_wait_and_run_source_code` or `jerry_debugger_run_client_source_code`?